mmdet/models/seg_heads/panoptic_fpn_head.py

```diff
@@ -0,0 +1,111 @@
+import torch
+import torch.nn as nn
+from mmcv.runner import ModuleList
+
+from ..builder import HEADS
+from ..utils import ConvUpsample
+from .base_semantic_head import BaseSemanticHead
+
+
+@HEADS.register_module()
+class PanopticFPNHead(BaseSemanticHead):
```

Comment:
add `get_xxx` to this head to make the interface more general

Reply:
We can discuss firstly.